CHANGELOG.md

```diff
@@ -31,13 +31,17 @@
 - Option completion for `apt list` now works properly (#4350).
 - Added completions for:
   - `as` (#4130)
+  - `bd`
```

Comment:
You can cite the number of this PR. (I understand you'd be hard-pressed to figure it out before submitting!)

Reply:
Ah thanks I will add the information.